app/concepts/matestack/ui/core/collection/content/content.rb

```diff
@@ -0,0 +1,18 @@
+module Matestack::Ui::Core::Collection::Content
+  class Content < Matestack::Ui::Core::Component::Dynamic
+
+    def setup
+      @rerender = true
+      @component_config = @component_config.except(:data, :paginated_data)
+    end
+
+    def response
+      components {
+        div do
```

Comment:
Can this `div` be savely removed?

Reply:
nope. otherwise vue.js will complain as this is a dynamic component template